core/lib/tasks/core.rake

```diff
@@ -187,9 +187,9 @@ use rake db:load_file[/absolute/path/to/sample/filename.rb]}
 end
 
 namespace :core do
-  desc 'Set "active" status on draft products where available_on is in the past'
```

Comment:
@kgorazd we also need to add in a migration adding make_active_at to set the active status for existing products that were available, and set status archived for discontinued products

Reply:
done